Code/wwnet/network-typedefs.h

```diff
@@ -23,9 +23,11 @@ typedef int SOCKET;
 #define WSAEALREADY      EALREADY
 #define WSAEADDRINUSE    EADDRINUSE
 #define WSAEADDRNOTAVAIL EADDRNOTAVAIL
+#define WSAEAFNOSUPPORT  EAFNOSUPPORT
```

Comment:
I think its probably worth merging this file into socket_wrapper.h and just have the one header encompassing the cross platform network ifdefs.